examples/intkey_go/src/sawtooth_intkey_client/main.go

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+package main
+
+import (
+	"fmt"
+	"github.com/hyperledger/sawtooth-sdk-go/logging"
+	flags "github.com/jessevdk/go-flags"
+	"os"
+)
+
+// All subcommands implement this interface
+type Command interface {
+	Register(*flags.Command) error
+	Name() string
+	KeyfilePassed() string
+	UrlPassed() string
+	Run() error
+}
+
+type Opts struct {
+	Verbose []bool `short:"v" long:"verbose" description:"Enable more verbose output"`
+	Version bool   `short:"V" long:"version" description:"Display version information"`
+}
+
+var logger *logging.Logger = logging.Get()
+
+func main() {
+	// TODO: Following if condition for version printing can be improved
+	arguments := os.Args[1:]
+	for _, arg := range arguments {
+		if arg == "-V" || arg == "--version" {
+			// TODO: Replace with distribution version from system
```

Comment:
TODO

Reply:
Done, version will be picked up during build